src/Utils/getUser.ts

```diff
@@ -1,7 +1,7 @@
-export function getUser(user: User | null | undefined): User | null {
+export function getUser(user?: User): User | undefined {
   let _user = user
 
-  if (_user === undefined) {
+  if (!_user) {
```

Comment:
Is this correct or did the code previously allowed for explicitly setting the value to `null`?